lib/websocket_proxy.c

```diff
@@ -0,0 +1,123 @@
+#include <stdio.h>
```

Comment:
Let's rename this file (and websocket_proxy.h) to tunnel.c (and .h), and let them concentrate on tunneling the data.
IMO it can be used for other purposes as well.